README_python.md

```diff
@@ -192,29 +192,50 @@ HALIDE_REGISTER_GENERATOR(MyFilter, my_filter)
 ```
 
 If you are using CMake, the simplest thing is to use
-`add_halide_library()` (defined in HalideGeneratorHelpers.cmake) with the `PYTHON_EXTENSION_LIBRARY` option:
+`add_halide_library` and `add_halide_python_extension_library()` (defined in HalideGeneratorHelpers.cmake):
 
 ```
+# Build a Halide library as you usually would, but be sure to include `PYTHON_EXTENSION`
 add_halide_library(my_filter
-                         GENERATOR my_filter_generator
-                         SOURCES my_filter_generator.cpp
-                         PYTHON_EXTENSION_LIBRARY
-                         [ FEATURES ... ]
-                         [ PARAMS ... ])
+                   GENERATOR my_filter_generator
+                   SOURCES my_filter_generator.cpp
+                   PYTHON_EXTENSION output_path_var
+                   [ FEATURES ... ]
+                   [ PARAMS ... ])
+
+# Now wrap the generated code with a Python extension.
+add_halide_python_extension_library(my_extension
+                                    MODULE_NAME my_module
+                                    HALIDE_LIBRARIES my_filter)
```

Comment:
I disagree here -- I think it's a not-unlikely use case that you might need to have the module-name be different from the target-name (to avoid collisions in CMake's target namespace), so making it explicit that this usage is a thing is worthwhile.